README.md

```diff
@@ -60,26 +66,26 @@ digging into the underlying \_\_ref\_\_erences:
 x <- 1:1e6
 y <- list(x, x, x)
 ref(y)
-#> █ [1:0x7f9adf16b078] <list> 
-#> ├─[2:0x7f9add3d9b48] <int> 
-#> ├─[2:0x7f9add3d9b48] 
-#> └─[2:0x7f9add3d9b48]
+#> █ [1:0x7fdb5fe92018] <list> 
+#> ├─[2:0x7fdb5e02f090] <int> 
+#> ├─[2:0x7fdb5e02f090] 
+#> └─[2:0x7fdb5e02f090]
 
 e <- rlang::env()
 e$self <- e
 ref(e)
-#> █ [1:0x7f9ada2fc7e8] <env> 
-#> └─self = [1:0x7f9ada2fc7e8]
+#> █ [1:0x7fdb60839ea8] <env> 
+#> └─self = [1:0x7fdb60839ea8]
 ```
 
 A related tool is `obj_size()`, which computes the size of an object
 taking these shared references into account:
 
 ``` r
 obj_size(x)
-#> 680 B
+#> 4,000,048 B
```

Comment:
@hadley I did this with a fresh install of lobstr from CRAN. I'm totally unclear as to why there's like a 6,000% increase in object size. 😞

Reply:
What version of R do you have?

Reply:
3.5.1

Reply:
Damn, I see it too. I better do a patch release 😦

Reply:
Argh no, it goes away if I install from CRAN with `type = "source"`

Reply:
OK, got it that's super weird.

Reply:
Ok, figured it out with Jim's help — I'll need to do a patch release.